src/commands/distribute/release.ts

```diff
@@ -284,13 +289,13 @@ export default class ReleaseBinaryCommand extends AppCommand {
     }
   }
 
-  private async distributeRelease(client: AppCenterClient, app: DefaultApp, releaseId: number, releaseNotesString: string): Promise<void> {
+  private async distributeRelease(client: AppCenterClient, app: DefaultApp, releaseId: number, releaseNotesString: string, notifyTesters: boolean): Promise<void> {
     await this.putReleaseDetails(client, app, releaseId, releaseNotesString);
     const distributionGroupResponse = await getDistributionGroup({
       client, releaseId, app: this.app, destination: this.distributionGroup, destinationType: "group"
     });
     await addGroupToRelease({
-      client, releaseId, distributionGroup: distributionGroupResponse, app: this.app, destination: this.distributionGroup, destinationType: "group", mandatory: false, silent: false
+      client, releaseId, distributionGroup: distributionGroupResponse, app: this.app, destination: this.distributionGroup, destinationType: "group", mandatory: false, silent: !notifyTesters
```

Comment:
This `silent` becomes `notifyTesters` again on the other side of this function call. Would you like this option name to change to maintain consistency?
https://github.com/microsoft/appcenter-cli/blob/master/src/commands/distribute/lib/distribute-util.ts#L52